frontend/src/metabase/dashboard/containers/AutomaticDashboardApp/AutomaticDashboardApp.tsx

```diff
@@ -179,7 +198,22 @@ class AutomaticDashboardAppInner extends Component {
                 </FixedWidthContainer>
               </div>
             )}
-            <Dashboard isXray {...this.props} />
+            {this.props.dashboard && (
+              <Dashboard
+                isXray
+                dashboard={this.props.dashboard}
+                selectedTabId={this.props.selectedTabId}
+                slowCards={this.props.slowCards}
+                clickBehaviorSidebarDashcard={null}
+                downloadsEnabled={false}
+                onEditingChange={_.noop}
+                autoScrollToDashcardId={undefined}
```

Comment:
Can we update the prop definition to `autoScrollToDashcardId?: DashCardId;` and omit it here?